daisy/compute/compute_test.go

```diff
@@ -288,8 +309,8 @@ func TestStops(t *testing.T) {
 	}
 	defer svr.Close()
 
-	stopURL = fmt.Sprintf("/%s/zones/%s/instances/%s/stop?alt=json&prettyPrint=false", testProject, testZone, testInstance)
-	opGetURL = fmt.Sprintf("/%s/zones/%s/operations//wait?alt=json&prettyPrint=false", testProject, testZone)
+	stopURL = fmt.Sprintf("/projects/%s/zones/%s/instances/%s/stop?alt=json&prettyPrint=false", testProject, testZone, testInstance)
+	opGetURL = fmt.Sprintf("/projects/%s/zones/%s/operations//wait?alt=json&prettyPrint=false", testProject, testZone)
```

Comment:
Why are these changed?

Reply:
Same as above, when the versions were updated in go.mod for the imported libraries these tests broke. It looked like the URL expectations changes from the library given the same input.